master/getting-started/kubernetes/upgrade/delete.md

```diff
@@ -3,5 +3,101 @@ title: Deleting old data
 no_canonical: true
 ---
 
-_Docs on this coming soon!_
+## About deleting old Calico data
 
+You may need or wish to manually delete Calico data from your etcd datastore under the
+following conditions.
+  
+- [You succeeded in migrating your data and upgrading to Calico v3.0. After
+  running Calico for some time and experiencing no errors, you want to delete
+  the old Calico data from the etcdv2 datastore](#deleting-calico-data-from-etcdv2-after-a-successful-migration-and-upgrade).
+  
+- [A data migration attempt failed partway through, leaving the etcdv3 datastore
+  with some, but not all of your etcvd2 data](#deleting-calico-data-from-etcdv3-after-a-partial-migration).
+  
+  
+## Deleting Calico data from etcdv2 after a successful migration and upgrade
+
+### Prerequisites
+
+- **etcdctl installed**: 
+
+  To install just etcdctl, [download the etcd release binary](https://github.com/coreos/etcd/releases), 
+  untar it, and extract the etcdctl binary.
+
+- **etcdctl configured to connect to your {{site.prodname}} etcdv2 datastore**: 
+
+  Use the options you [configured `calico-upgrade` with](/{{page.version}}/getting-started/kubernetes/upgrade/setup#configuring-calico-upgrade-to-connect-to-the-etcdv2-datastore)
+  for [etcdctl](https://github.com/coreos/etcd/blob/master/etcdctl/README.md).
+  
+
+### Deleting Calico data from etcdv2
+
+1. Issue the following command to retrieve a list of all of the Calico keys.
+   
+   ```
+   etcdctl --endpoints=<etcdv2-hostname:port> get / --prefix --keys-only
+   ```
+   
+1. Issue the following command to delete the {{site.prodname}} keys.
+   
+   ```
+   etcdctl --endpoints=<etcdv2-hostname:port> del / --prefix 
```

Comment:
I'm pretty sure for the v2 version of etcdctl, it's `--endnpoint` without the `s`.

If you specify `ETCDCTL_API=3` then it takes `--endpoints`